src/app/api/update-meetups/route.ts

```diff
@@ -53,34 +53,38 @@ export async function GET() {
   const communityRecords = await getCommunities();
   const eventRecords = await getEvents();
 
-  const communities = communityRecords.map(async (record) => {
-    const community = recordToCommunity(record);
+  const communities = communityRecords
+    .map(async (record) => {
+      const community = recordToCommunity(record);
 
-    const meetupEventsUrl = createUrl(community.meetup_url, 'events');
-    log(`Fetching meetup url ${meetupEventsUrl.toString()}`, community);
+      const meetupEventsUrl = createUrl(
+        community.meetup_url.replace('www.', 'api.'),
+        'events'
+      );
+      log(`Fetching meetup url ${meetupEventsUrl.toString()}`, community);
 
-    const response = await fetch(meetupEventsUrl);
+      const response = await fetch(meetupEventsUrl, { cache: 'no-store' });
```

Comment:
noticed the lack of cache directive here. I wonder if that was causing problems with the cheerio version of this